### PR TITLE
Remove community-translator feature flag

### PR DESCRIPTION
**Pull request by dllh**
This feature flag is enabled in all environments and thus is no longer needed. The feature is also enabled on a per-user basis via settings, so it should be pretty safe to remove the flags.

I'm not really sure how to test. I inherited this clean-up task but don't know anything about the feature or how to validate that it's working as expected. I'm hoping somebody from our i18n team can help fill out the testing section here, and maybe help confirm that nothing's broken.

See p4TIVU-5GO-p2

### CODE REVIEW

**Comment by rachelmcr** [comment]
@dllh I came across this PR while reviewing pending PRs; it looks like it was never merged and has some conflicts. Could you take a look?

**Comment by akirk** [comment]
@rachelmcr I'll take over responsibility for this one.

**Comment by Copons** [comment]
Maybe too many feature flags removed? 😄

File: config/development.json [hunk 1]
```diff
@@ -37,13 +37,8 @@
 		"ad-tracking": false,
 		"automated-transfer": true,
 		"apple-pay": true,
+		"boosted-post-survey": true,
 		"code-splitting": true,
-		"comments/filters-in-posts": true,
-		"comments/moderation-tools-in-posts": true,
-		"comments/management": true,
-		"comments/management/all-list": false,
-		"manage/comments/bulk-actions": false,
```
Comment (dmsnell) [inline config/development.json]: whoops. removed too many flags here…

Comment (gwwar) [inline config/development.json]: Please be careful when removing flags that are not related to the PR @dllh @akirk

File: config/development.json [hunk 2]
```diff
@@ -37,13 +37,8 @@
 		"ad-tracking": false,
 		"automated-transfer": true,
 		"apple-pay": true,
+		"boosted-post-survey": true,
```
Comment (gwwar) [inline config/development.json]: Did we mean to add this?

**Comment by akirk** [comment]
Sorry! The diff was clean upon rebase :( Launching a new PR.
